common/reviews/api/rush-lib.api.md

```diff
@@ -904,6 +944,14 @@ export class RushUserConfiguration {
     static initializeAsync(): Promise<RushUserConfiguration>;
 }
 
+// Warning: (ae-incompatible-release-tags) The symbol "_setBuildTimes" is marked as @beta, but its signature references "IExecutionResult" which is marked as @alpha
```

Comment:
To fix this warning, you could just promote `IExecutionResult` to be `@beta`.  The `@alpha` vs `@beta` distinction is not very meaningful for `rush-lib`